src/functionalTest/resources/scenarios/RIA-6682-send-appeal-submitted-with-remission-pa-appeal-type-notification.json

```diff
@@ -1,20 +1,19 @@
 {
-  "description": "RIA-3689 send appeal submitted with remission and payOffline, for PA appeal notification",
+  "description": "RIA-6682 send appeal submitted with remission and payOffline, for PA appeal notification",
```

Comment:
If FT was against 3689 originally maybe file and desc should be called "RIA-3689-RIA-6682". Ive seen this done in other FTs. However its a really small point you can just ignore this comment as it doesnt affect the actual test. More for tracking/auditing purposes really.